### PR TITLE
Vet clinic database: add "join table" for visits

### PR DESCRIPTION
**Pull request by degisew**
### Vet clinic database: add "join table" for visits
**In this PR, I implemented the following features:**

[shoot1](https://github.com/degisew/Vet-clinic-database/blob/join-table/shoots/lq1.PNG)

[shoot2](https://github.com/degisew/Vet-clinic-database/blob/join-table/shoots/lq2.PNG)

[shoot3](https://github.com/degisew/Vet-clinic-database/blob/join-table/shoots/lq3.PNG)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,9 +41,18 @@ Give a ⭐ if you like this project!
 
 This project is [MIT](./MIT.md) licensed.
```
Comment (1GORDON) [inline README.md]: Great job @degisew ✋

- [ ] Kindly use the right readme file for your projects. To get started, you may use this [readme template](https://github.com/microverseinc/readme-template) provided by Microverse. You may also fix the broken MIT license link to make your readme file more professional.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @degisew ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (degisew) [reply README.md]: Hello @1GORDON 
Thanks for your comment. I have one question for you. How can I fix the broken MIT license on README?

Comment (peterrobert) [reply README.md]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @degisew, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**